src/components/Notification/Notification.stories.mdx

```diff
@@ -118,11 +131,22 @@ be rendered inside other landmark elements such as the `header` and `footer`
 landmark elements. Adding a `Notification` component inside an HTML `main` landmark
 element is acceptable.
 
+Every `Notification` component must have a unique `aria-label` attribute. This
+is set through the `ariaLabel` prop. A unique `aria-label` value, along with the
+`aside` HTML landmark element, helps screen readers better navigate a page with
+multiple notification content.
```

Comment:
"multiple notification content" sounds a little strange to me – maybe just "multiple notifications" or "multiple notification components"?